.github/workflows/lint.yaml

```diff
@@ -0,0 +1,23 @@
+name: Lint
+on:
+  pull_request:
+  push:
+    branches: [master]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v2.4.0
+        with:
+          node-version: 14.x
+          registry-url: https://registry.npmjs.org
+          cache: yarn
+      - run: yarn install --frozen-lockfile --check-files
+
+      - uses: pre-commit/action@v2.0.3
```

Comment:
would just be easiest to add 

```
    - uses: actions/setup-python@v2
    - uses: pre-commit/action@v2.0.3
```

to the main pipeline since this linting is already done in the main pipeline